strider/traversal.py

```diff
@@ -0,0 +1,60 @@
+"""Traversal."""
```

Comment:
This isn't very helpful.

Reply:
77c3964a535622072698e8be3c438ec4ab04dae6